scripts/mutect2_wdl/mutect2.json

```diff
@@ -17,5 +17,6 @@
   "Mutect2.dbsnp_index": "$__dbsnp_index__",
```

Comment:
I really like that you made two json templates.  One for single pair and one for multi-pair.